revscoring/features/wikitext/datasources/tokenized.py

```diff
@@ -0,0 +1,437 @@
+        """
+
+        self.url_prop_delta = frequencies.prop_delta(
+            self.revision.parent.url_frequency,
+            self.url_delta,
+            name=self._name + ".url_prop_delta"
+        )
+        """
+        A url proportional frequency delta table
+        """
+
+        self.word_delta = frequencies.delta(
+            self.revision.parent.word_frequency,
+            self.revision.word_frequency,
+            name=self._name + ".word_delta"
+        )
+        """
+        A lower-cased word frequency delta table
+        """
+
+        self.word_prop_delta = frequencies.prop_delta(
+            self.revision.parent.word_frequency,
+            self.word_delta,
+            name=self._name + ".word_prop_delta"
+        )
+        """
+        A lower-cased word proportional frequency delta table
+        """
+
+        self.uppercase_word_delta = frequencies.delta(
+            self.revision.parent.uppercase_word_frequency,
+            self.revision.uppercase_word_frequency,
+            name=self._name + ".uppercase_word_delta"
+        )
+        """
+        A uppercase word frequency delta table
+        """
+
+        self.uppercase_word_prop_delta = frequencies.prop_delta(
+            self.revision.parent.uppercase_word_frequency,
+            self.uppercase_word_delta,
+            name=self._name + ".uppercase_word_prop_delta"
+        )
+        """
+        A uppercase word proportional frequency delta table
+        """
+
+        self.punctuation_delta = frequencies.delta(
+            self.revision.parent.punctuation_frequency,
+            self.revision.punctuation_frequency,
+            name=self._name + ".punctuation_delta"
+        )
+        """
+        A punctuation frequency delta table
+        """
+
+        self.punctuation_prop_delta = frequencies.prop_delta(
+            self.revision.parent.punctuation_frequency,
+            self.punctuation_delta,
+            name=self._name + ".punctuation_prop_delta"
+        )
+        """
+        A punctuation proportional frequency delta table
+        """
+
+        self.break_delta = frequencies.delta(
+            self.revision.parent.break_frequency,
+            self.revision.break_frequency,
+            name=self._name + ".break_delta"
+        )
+        """
+        A break frequency delta table
+        """
+
+        self.break_prop_delta = frequencies.prop_delta(
+            self.revision.parent.break_frequency,
+            self.break_delta,
+            name=self._name + ".break_prop_delta"
+        )
+        """
+        A break proportional frequency delta table
+        """
+
+
+def is_uppercase_word(word_token):
+    return len(word_token) > 1 and \
+           sum(c.lower() != c for c in word_token) == len(word_token)
+
+
+class TokenIsInTypes:
+
+    def __init__(self, types):
+        self.types = set(types)
+
+    def filter(self, token):
+        return token.type in self.types
+
+
+def _process_tokens(text):
```

Comment:
I looked into this and it seems messy at best.  I think it's more simplistic to leave it as a module-level protected function.